README.md

```diff
@@ -1,15 +1,19 @@
 [![Build Status](https://secure.travis-ci.org/jaysonsantos/django-bmemcached.png?branch=master)](http://travis-ci.org/jaysonsantos/django-bmemcached)
+
 # Django-BMemcached
-A django cache backend to use [bmemcached] (https://github.com/jaysonsantos/python-binary-memcached) module which supports memcached binary protocol with authentication.
+
+A django cache backend to use [bmemcached](https://github.com/jaysonsantos/python-binary-memcached) module which supports memcached binary protocol with authentication.
```

Comment:
Removed rogue space between `[...]` and `(...)` that meant that the link wasn't rendering correctly.